src/lithium/interestingness/repeat.py

```diff
@@ -0,0 +1,81 @@
+
+"""Repeats an interestingness test a given number of times. If "REPEATNUM" is present, it is replaced in turn with each
+number as it increments. This "REPEATNUM" can be customized if required.
+
+Use for:
+
+1. Intermittent testcases.
+
+   Repeating the test can make the bug occur often enough for Lithium to make progress.
+
+    python -m lithium repeat 20 crashes --timeout=9 ./js --fuzzing-safe testcase.js
+
+2. Unstable testcases.
+
+   Varying a number in the test (using REPEATNUM) may allow other parts of the testcase to be
+   removed (Lithium), or may allow different versions of the shell to crash (autoBisect).
+
+   In the testcase:
+     schedulegc(n);
+
+   On the command line:
+     python -m lithium repeat 20 crashes --timeout=9 ./js --fuzzing-safe -e "n=REPEATNUM;" testcase.js
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import logging
+import sys
+
+from .utils import rel_or_abs_import
+
+
+def interesting(cli_args, temp_prefix):
+    """Interesting if the desired interestingness test that is run together with "repeat" also reports "interesting".
+
+    Args:
+        cli_args (list): List of input arguments.
+        temp_prefix (str): Temporary directory prefix, e.g. tmp1/1 or tmp4/1
+
+    Returns:
+        bool: True if the desired interestingness test also returns True, False otherwise.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--REPEATNUM", default="REPEATNUM", dest="repeat_num",
+                        help="Set the cookie that is to be altered in the testcase. Defaults to '%(default)s'.")
+    parser.add_argument("cmd_with_flags", nargs=argparse.REMAINDER)
+    args = parser.parse_args(cli_args)
+
+    logger = logging.getLogger(__name__)  # __name__ should be lithium.interestingness.repeat
+    logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+    logging.getLogger("flake8").setLevel(logging.WARNING)
+    # logger_handler = logging.StreamHandler(sys.stdout)
+    # logger_handler.setLevel(logging.DEBUG)
+    # logger_handler.terminator = ""  # We cannot yet suppress the ending newline yet, until Python 2.7 is deprecated
+    # logger.addHandler(logger_handler)
+
+    loop_num = int(args.cmd_with_flags[0])
+    assert loop_num > 0, "Mininum number of iterations should be at least 1"
+
+    condition_script = rel_or_abs_import(args.cmd_with_flags[1])
+    condition_args = args.cmd_with_flags[2:]
+
+    if hasattr(condition_script, "init"):
+        condition_script.init(condition_args)
+
+    # Run the program over as many iterations as intended, with desired flags, replacing REPEATNUM where necessary.
+    for i in range(loop_num):
+        # This doesn't do anything if REPEATNUM is not found.
+        replaced_condition_args = [s.replace("REPEATNUM", str(i)) for s in condition_args]
+        logger.info("Repeat number %d:", i + 1)  # Ending newline not suppressed yet until Python 2.7 is deprecated
```

Comment:
Previously with version 0.1.0, it used to print something like:

```
Repeat number 1: Exit status: NORMAL (0.184 seconds)
Repeat number 2: Exit status: NORMAL (0.224 seconds)
Repeat number 3: Exit status: NORMAL (0.207 seconds)
Repeat number 4: Exit status: NORMAL (0.086 seconds)
Repeat number 5: Exit status: NORMAL (0.064 seconds)
```

Now it will print:

```
INFO:lithium:Repeat number 1:
INFO:lithium:Exit status: NORMAL (0.184 seconds)
INFO:lithium:Repeat number 2:
INFO:lithium:Exit status: NORMAL (0.224 seconds)
INFO:lithium:Repeat number 3:
INFO:lithium:Exit status: NORMAL (0.207 seconds)
INFO:lithium:Repeat number 4:
INFO:lithium:Exit status: NORMAL (0.086 seconds)
INFO:lithium:Repeat number 5:
INFO:lithium:Exit status: NORMAL (0.064 seconds)
```

I guess it isn't much of an issue.